Code/Source/RobotControl/ROS2RobotControlComponent.cpp

```diff
@@ -73,5 +73,17 @@ namespace ROS2
     {
         // TODO - also, dependent on current/selected RobotControl implementation for what components are required
         required.push_back(AZ_CRC("ROS2Frame"));
+        required.push_back(AZ_CRC("PhysicsRigidBodyService"));
```

Comment:
This is not the case now

Reply:
remove TODO instead, don't add this. This is a dependency for RigidBodyController, not RobotController